_data/wai-course-list/submissions/11fef4d0-f91e-11ed-82d9-1d8727814408.json

```diff
@@ -0,0 +1,88 @@
+{
+  "accessibility_support": [
+    "general-accessibility-material-ahead-of-time",
+    "general-accessibility-pre-check-available",
+    "digital-accessibility-caption-audio",
+    "digital-accessibility-text-and-visuals-can-be-resized",
+    "digital-accessibility-sufficient-contrast",
+    "digital-accessibility-motions-or-animations",
+    "digital-accessibility-description-of-visual-information",
+    "digital-accessibility-all-content-meets-wcag",
+    "non-digital-accessibility-accessible-facility",
+    "non-digital-accessibility-assistive-listening-devices",
+    "non-digital-accessibility-text-and-visuals-big-enough",
+    "non-digital-accessibility-sufficient-contrast",
+    "non-digital-accessibility-description-of-visual-information"
+  ],
+  "accessibility_support_other": "This training can be customized according to your accessibility needs. Some adjustments may induce additional costs as they are provided by third-party providers.",
+  "audience": [
+    "audience_content_author",
+    "audience_designer",
+    "audience_general"
+  ],
+  "check_correct_info": "on",
+  "check_publish_info": "on",
+  "check_scope_info": "on",
+  "comments": "",
```

Comment:
Can we had comments : "On-site teaching sessions can be scheduled upon request."

Thank you !

Reply:
We can add this comment, however this information **will not** displayed in the list - as it is used for internal comments only. 
In the submission form it is indicated that any information included there **will not** be displayed on the Course List, but will be publicly available since all data related to this resource is stored on a public/open repository on GitHub.

If you would like to update the course description to include this information, please let me know which details you would like to be displayed.